assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -14,6 +14,8 @@
     "MIT License\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import unittest
</pre><p>Please revert this. Each PR should only focus on one thing. You can add UT in a later PR. Also, the format here is incorrect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>